### PR TITLE
sshs: Add version 4.5.1

### LINKED ISSUE #13003: [Request]: sshs
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

sshs

### Description

Terminal user interface for SSH

### Homepage

https://github.com/quantumsheep/sshs

### Download Link(s)

https://github.com/quantumsheep/sshs/releases/latest/download/sshs-windows-amd64.exe

### Some Indication of Popularity/Repute

Included in Terminal Trove
https://terminaltrove.com/sshs/

### PR DESCRIPTION
**Pull request by xd-mau5**
Closes #13003

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/sshs.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "4.5.1",
+    "description": "Terminal user interface for SSH",
+    "homepage": "https://github.com/quantumsheep/sshs",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/quantumsheep/sshs/releases/download/4.5.1/sshs-windows-amd64.exe",
```
Comment (aliesbelik) [inline bucket/sshs.json:8]:
```suggestion
            "url": "https://github.com/quantumsheep/sshs/releases/download/4.5.1/sshs-windows-amd64.exe#/sshs.exe",
```

File: bucket/sshs.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            "hash": "3a4feeeb944962021281b05ecf0226a046ecd6f12bc1fe36d4d74b77b202c6b8"
+        },
+        "arm64": {
+            "url": "https://github.com/quantumsheep/sshs/releases/download/4.5.1/sshs-windows-arm64.exe",
```
Comment (aliesbelik) [inline bucket/sshs.json:12]:
```suggestion
            "url": "https://github.com/quantumsheep/sshs/releases/download/4.5.1/sshs-windows-arm64.exe#/sshs.exe",
```

File: bucket/sshs.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    ],
+    "bin": "sshs.exe",
+    "checkver": {
+        "github": "https://github.com/quantumsheep/sshs",
+        "regex": "releases/download/(\\d+\\.\\d+\\.\\d+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/quantumsheep/sshs/releases/download/$version/sshs-windows-amd64.exe"
```
Comment (aliesbelik) [inline bucket/sshs.json:28]:
```suggestion
                "url": "https://github.com/quantumsheep/sshs/releases/download/$version/sshs-windows-amd64.exe#/sshs.exe"
```

File: bucket/sshs.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            },
+            "arm64": {
+                "url": "https://github.com/quantumsheep/sshs/releases/download/$version/sshs-windows-arm64.exe"
```
Comment (aliesbelik) [inline bucket/sshs.json:31]:
```suggestion
                "url": "https://github.com/quantumsheep/sshs/releases/download/$version/sshs-windows-arm64.exe#/sshs.exe"
```

File: bucket/sshs.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        },
+        "arm64": {
+            "url": "https://github.com/quantumsheep/sshs/releases/download/4.5.1/sshs-windows-arm64.exe",
+            "hash": "9afdde6b3b0c84a0d22729ceef7cce9e8d9c4360a8755fdf9d0bfb1b4067e1c6"
+        }
+    },
+    "pre_install": [
+        "if ($architecture -eq '64bit') { Rename-Item \"$dir/sshs-windows-amd64.exe\" \"sshs.exe\" }",
+        "if ($architecture -eq 'arm64') { Rename-Item \"$dir/sshs-windows-arm64.exe\" \"sshs.exe\" }"
+    ],
```
Comment (aliesbelik) [inline bucket/sshs.json:19]:
```suggestion
    },
```

File: bucket/sshs.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    "bin": "sshs.exe",
+    "checkver": {
+        "github": "https://github.com/quantumsheep/sshs",
+        "regex": "releases/download/(\\d+\\.\\d+\\.\\d+)"
+    },
```
Comment (aliesbelik) [inline bucket/sshs.json:24]:
```suggestion
    "checkver": "github",
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
- Inline renaming via `#/` is more laconic and makes `pre_install` unnecessary.
- Default regex for github link in `checkver` is enough.

**Comment by aliesbelik** [comment]
Looks like could be promoted into the `Main` bucket instead, since fits the [Main criteria](https://github.com/ScoopInstaller/Scoop/wiki/Criteria-for-including-apps-in-the-main-bucket).

**Comment by xd-mau5** [comment]
Should i make the pull request in Main then?

**Comment by aliesbelik** [comment]
> Should i make the pull request in Main then?

I'm not maintainer of Main so I couldn't transfer your request and pr, but if you are interested in promoting requested tool into Main instead Extras, I suppose this would be a more suitable place.
Otherwise could be merged into Extras, with the requested changes.

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**